less/angular/pages/private-eye.less

```diff
@@ -1,44 +1,4 @@
-.private-eye-footer {
-  text-align: center;
+.cta-footer {
```

Comment:
It's possible, but I kind of wanted to leave it segregated since it's very much so page specific. There shouldn't be much of a performance hit by having the extra file there.